src/Identity/test/Identity.Test/IdentityUIScriptsTest.cs

```diff
@@ -148,13 +146,17 @@ private static List<ScriptTag> GetScriptTags(string cshtmlFile)
         private static string GetSolutionDir()
         {
             var dir = new DirectoryInfo(AppContext.BaseDirectory);
-            while (dir != null)
+            // On helix we use the published copy
+            if (!string.Equals(Environment.GetEnvironmentVariable("helix"), "true", StringComparison.OrdinalIgnoreCase))
```

Comment:
I would rather prefer we had a shared way of doing this (looking for the test base directory). Rather then copying the snippet around.

Reply:
Do you mean add a test helper method for this that both the helix skip attribute and actual tests can use to check if we are running on helix?

Reply:
Add a way to find the test asset root. Or at minimum to detect running in helix.

Reply:
I don't feel comfortable generalizing a single test, I can add the helix detection now since the HelixSkip has the same logic.  Once I fix a few more tests and see a clear pattern we can look into a generalized shared GetSolutionDir.

Reply:
@pakrym added a static helper method for now which encapsulates the Environment helix check